File: quickactionview/src/main/java/com/commit451/quickactionview/ColorUtils.java
package com.commit451.quickactionview;

import android.content.Context;
import android.util.TypedValue;

/**
 * Gets the colors
 */
class ColorUtils {

    private static TypedValue sTypedValue = new TypedValue();

    public static int getThemeAttrColor(Context context, int attributeColor) {
        context.getTheme().resolveAttribute(attributeColor, sTypedValue, true);
        return sTypedValue.data;
    }
}

File: quickactionview/src/main/java/com/commit451/quickactionview/ColorUtils.kt
package com.commit451.quickactionview

import android.content.Context
import android.util.TypedValue

/**
 * Gets the colors
 */
internal object ColorUtils {

    private val typedValue = TypedValue()

    fun getThemeAttrColor(context: Context, attributeColor: Int): Int {
        context.theme.resolveAttribute(attributeColor, typedValue, true)
        return typedValue.data
    }
}
